### DISCUSSION TITLE
Notifiche mancanti per annotazioni di Rettifica o Errore materiale, che comportino aggiornamenti in Anagrafe.

### DISCUSSION THREAD

**Discussion by gcuci** [italia/ansc#962] [Miglioramenti] [answered]
Buongiorno,

riportiamo una segnalazione dal Comune di Firenze.

Le annotazioni di Rettifica o Errore materiale, in base alle informazioni che vengono modificate, possono comportare anche un aggiornamento in Anagrafe. Nel nuovo processo ANSC, però, non sembra siano previste eventuali notifiche per Anagrafe, che permettano di effettuare questo aggiornamento. E' un'integrazione non ancora disponibile, o non è stata presa in considerazione?

In attesa di riscontro,

cordiali saluti.

**Comment by gbuemi** [comment]
Salve @gcuci, 
per le operazioni che comportano un aggiornamento in anagrafe dovrebbero essere generate le notifiche. Potete fornirci per favore gli id degli atti per il quale non sono state generate? Grazie

Un saluto,
Supporto ANSC

**Comment by gcuci** [comment]
Potete far riferimento all'atto di Nascita (idAnsc: 2024-3013781-56-039002), validato e firmato, e per il quale è stata inviata la data validità.

Dopo aver confermato la Notifica di Trascrizione ricevuta, nella Web App di ANPR, troviamo la relativa Notifica. 

Sullo stesso atto, è stata successivamente validata e firmata, un'annotazione di Rettifica per Errore (idAnsc: 2024-3013921-18-039002), con la modifica del nome dell'intestatario. Per questa però, non viene generata alcuna Notifica in ANSC nè in ANPR.

E' forse necessario, anche in questo caso, per attivare la Notifica da ANPR, inviare la data validità con l'apposito servizio?

**Comment by gbuemi** [comment, marked as answer]
Salve @gcuci,
confermiamo che anche in questo caso è necessario inviare la data validità con apposito servizio.

Un saluto,
Supporto ANSC
